rfcs/README.md

```diff
@@ -1,48 +1,123 @@
+   discussion via inline comments.
+4. Start with "why". Lead with one or two sentences describing the higher level intent. Leading with "how" omits the
+   opportunity to consider alternative approaches.
+
+### Keep the scope small
+
+Use the "Scope" section to address out of scope concerns, like future improvements. This signals to the team that
+you are aware of these concerns but have explicitly chosen to defer them. This not only helps to keep the RFC
+discussion focused, but also the implementation, resultin in a quicker overall delivery time.
+
+### Be opinionated and propose a single solution
 
-### Before creating a RFC
+Your job as the RFC author is to navigate the issue at hand and propose an opinionated solution. Ambiguity creates
+unproductive discussion and should be eliminated while drafting the RFC. This does not mean you need to go at it alone.
+You are encouraged to investigate and discuss solutions with your team while you draft the RFC, but the end state of
+the RFC should land on a single recommendation that you are reasonably confident in. Use the "Rationale" section to
+justify your proposal and the "Alternatives" section to note other solutions you considered. See the [FAQ](#faq) for
+more info navigating your solution.
 
-1. Search Github for previous issues and RFCs on this topic.
-2. Open an issue representing the change for light discussion.
-3. In the issue, obtain a consensus that an RFC is necessary.
-   * The change might get quickly rejected.
+## Process
+
+### Before creating an RFC
+
+1. Search Github for [previous issues](https://github.com/timberio/vector/issues) and
+   [RFCs](https://github.com/timberio/vector/tree/master/rfcs) on this topic.
+1. If an RFC issue does not exist, [open one](https://github.com/timberio/vector/issues/new/choose).
+1. Use the issue to obtain consensus that an RFC is necessary.
+   * The change might be quickly rejected.
    * The change might be on our long term roadmap and get deferred.
    * The change might be blocked by other work.
 
-### Creating a RFC
+### Creating an RFC
+
+1. Create a new branch
+1. Copy the [`rfcs/_YYYY-MM-DD-issue#-title.md`](rfcs/_YYYY-MM-DD-issue%23-title.md) template with the appropriate
+   name. Be sure to use the issue number you created above. (e.g., `rfcs/2020-02-10-445-internal-observability.md`)
+1. Fill in your RFC, pay attention the bullets and guidelines. Do not ommit any sections.
+1. Work with your team to land on a confident solution. Allocate time for code-level spikes if necessary.
+1. Submit your RFC as a pull request and tag reviewers for approval.
+
+### Accepting an RFC
+
+1. At least 3 team members must approve your RFC in the form of pull request approvals.
+1. Once approved, self-merge your RFC.
+
+### Implementing an RFC
+
+1. Create issues from the "Plan Of Attack" section. Place them in an epic if necessary.
+1. Coordinate with leadership to schedule your work.
```

Comment:
This too doesn't fully apply to open-source contributors.

I'm fine leaving it as is, but these _are_ steps that non-DD employees can't necessarily take.